### PR TITLE
Add LICENSE

### LINKED ISSUE #7: Add license
Needs `license` property in `package.json` and a license file.

### PR DESCRIPTION
**Pull request by zkochan**
close #7

### CODE REVIEW

**Comment by zkochan** [comment]
@styfle any chance having this merged? It is important for some of our pnpm users to have all deps with license

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (styfle) [inline LICENSE]:
```suggestion
The MIT License (MIT)
```

**Comment by styfle** [comment]
It looks like `now-cli` uses apache but nearly all other ZEIT projects use MIT so I think MIT is a good choice for `fetch-retry`.

File: LICENSE (hunk 1 above)
Comment (styfle) [reply LICENSE]: This was the only inconsistency I found with our other project licenses

Comment (zkochan) [reply LICENSE]: hm, OK, I copied the license from here https://github.com/zeit/fetch/blob/master/LICENSE

**Pull request merged**